tests/apollo/test_skvbc_ro_replica.py

```diff
@@ -152,7 +152,6 @@ async def test_ro_replica_start_simultaneously (self, bft_network, tracker):
                                 log.log_message(message_type="Replica" + str(ro_replica_id) + " : lastExecutedSeqNum:" + str(lastExecutedSeqNum))
                                 nursery.cancel_scope.cancel()
 
-    @unittest.skip("Unstable test - BC-17828")
```

Comment:
Didn't you say that you were using retries for these tests?
I can't find it in the code.

Reply:
No, I just uncommented the skip pragmas. Now the tests pass